src/net/inv.rs

```diff
@@ -178,10 +178,10 @@ impl PeerBlocksInv {
 
             let set_idx = (insert_index / 8) as usize;
             let set_bit = insert_index % 8;
-            if set_idx >= self.block_inv.len() {
+            while set_idx >= self.block_inv.len() {
```

Comment:
This could use resize() rather than iterating push()

Reply:
Fixed in 28f4b7f